crates/ruff/src/rules/flake8_use_pathlib/rules/os_path_getatime.rs

```diff
@@ -0,0 +1,42 @@
+use ruff_diagnostics::Violation;
+use ruff_macros::{derive_message_formats, violation};
+
+/// PTH203
+
+/// ## What it does
+/// Detects the use of `os.path.getatime`.
+///
+/// ## Why is this bad?
+/// `pathlib` offers high-level path manipulations of paths, `os` offers low-level manipulation of paths.
```

Comment:
nit (and kinda  annoying since it's not automated, sorry): The rule description doc comment are generally wrapped at 80 characters